common/src/main/java/com/wynntils/utils/colors/CommonColors.java

```diff
@@ -1,10 +1,13 @@
 /*
- * Copyright © Wynntils 2022.
+ * Copyright © Wynntils 2022-2023.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
 package com.wynntils.utils.colors;
 
 public final class CommonColors {
+    // Note: Rendering this is not supported, except in BeaconBeamFeature
+    public static final CustomColor RAINBOW = CustomColor.fromInt(0xf0f0f0).withAlpha(0);
```

Comment:
... a tad hacky, yes? 🤔

Reply:
This is actually rather easy to unhack. We change this to a supplier, and just use the rainbow generation method from beacon beam feature. 1 problem is that there is no place for that event.. otherwise, voila, we have a generic rainbow.

Reply:
You mean like a CustomColorSupplier..?

Reply:
Just a `Supplier<CustomColor`. For this only case. We would get a new value every render call.

Reply:
I'm not sure I follow you completely where and how this supplier should be used, but if you think you can easily un-hack this then I'm okay with this for now.